apps/micro_react/src/app/app.tsx

```diff
@@ -6,16 +6,17 @@ import { Banner, RatingStar } from '@micro-frontend-react/common-ui';
 import { Characters } from '../components/characters';
 
 const App = () => {
-  const [showCharacters, setShowCharacters] = useState(false);
+  const [showCharacters, setShowCharacters] = useState<boolean>(false);
   const handleClick = () => setShowCharacters(!showCharacters);
   return (
     <div className={styles['app_wrapper']}>
       <NxWelcome title="Micro_react App" />
       <Banner
         handleClick={handleClick}
         showCharacters={showCharacters}
-        children={!showCharacters ? 'Show characters' : 'Hide'}
```

Comment:
Instead of passing `children` through props, it's more readable to do it like this:

```
<Banner>
        {!showCharacters ? 'Show characters' : 'Hide'}
</Banner>
```